stylesheets/components/snackbar.css

```diff
@@ -0,0 +1,54 @@
+@import url("/stylesheets/main.css");
+
+.snackbar-wrapper {
+  max-width: 25rem;
+  background-color: var(--bg-black);
+  padding: 0.8rem;
+  margin: 0.5rem;
+  box-shadow: var(--shadow);
+}
+
+.snackbar-icon {
+  margin-left: auto;
+  color: var(--bg-white);
+}
+
+.snackbar-basic {
+  display: flex;
+  align-items: center;
+}
+
+.snackbar-btn {
+  text-transform: uppercase;
+  background-color: transparent;
+  font-size: 0.9rem;
+  color: var(--oc-primary);
+  padding: 0.2rem;
+  margin: 0.4rem;
+}
+
+.snackbar-btn:hover {
+  color: var(--oc-primary-2);
+}
+
+.snackbar-text {
+  color: var(--bg-white);
+  font-size: 0.9rem;
+}
+
+.snackbar-wrapper-2 {
+  display: flex;
+  align-items: center;
+  margin-left: auto;
+}
+
+.snackbar-stacked-wrapper {
```

Comment:
just suggestion not mandatory but you can name it as snackbar-stack-wrap like main aim is to shorten the class name it's up to you wanted it or not just suggested
This is also fine no problem with it.

Reply:
Will keep that in mind.